spec/namespaces/cluster.yaml

```diff
@@ -861,9 +859,6 @@ components:
     cluster.delete_component_template::query.master_timeout:
       in: query
       name: master_timeout
-      description: |-
-        Period to wait for a connection to the cluster-manager node.
-        If no response is received before the timeout expires, the request fails and returns an error.
       deprecated: true
```

Comment:
@Naarcha-AWS Why were these descriptions for `cluster_manager_timeout`, `master_timeout` & `timeout` removed?